xapi-compression.opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
```

Comment:
This should be in Opam 2 format.

Reply:
There were a few opam 2-isms missing from the new .opam files - I've updated both files.